### PR TITLE
example awsIntegrationUpdate

### PR DESCRIPTION
**Pull request by ianroy-opslevel**

### CODE REVIEW

File: graphql_query_examples/graphql_mutation_examples.md [hunk 1]
```diff
@@ -143,76 +245,6 @@ mutation enable_badge_for_service{
 ```
 </details>
 
-### 🧬 groupCreate (parent group, top level group)
```
Comment (eapache-opslevel) [inline graphql_query_examples/graphql_mutation_examples.md:146]: did you intend to delete these lines?

Comment (ianroy-opslevel) [reply graphql_query_examples/graphql_mutation_examples.md:146]: Yes, groups are deprecated now

File: graphql_query_examples/graphql_mutation_examples.md [hunk 2]
```diff
@@ -54,6 +54,108 @@ mutation aliasDelete_for_a_service{
 }
 ```
 
+### 🧬 awsIntegrationUpdate, get AWS integrations first, then reactivate integration for next scheduled sync
+
+Note: `integrationReactivate` will reactivate the integration and OpsLevel will retry syncing resources on the next scheduled sync.
```
Comment (eapache-opslevel) [inline graphql_query_examples/graphql_mutation_examples.md:59]: maybe worth mentioning that this is only needed if the integration was invalidated (per the `invalidatedAt` field)?

**Review by eapache-opslevel — APPROVED** [review]
a few small questions, but mostly LGTM

File: graphql_query_examples/graphql_mutation_examples.md [hunk 3]
```diff
@@ -54,6 +54,108 @@ mutation aliasDelete_for_a_service{
...
+
+```graphql
+query integrations_aws {
```
[the remark this answers is not in the document]
Comment (ianroy-opslevel) [reply graphql_query_examples/graphql_mutation_examples.md:62]: Going to leave it as is.

File: graphql_query_examples/graphql_mutation_examples.md (hunk 2 above)
Comment (ianroy-opslevel) [reply graphql_query_examples/graphql_mutation_examples.md]: Updated to call this out!

**Comment by ianroy-opslevel** [comment]
@eapache-opslevel Made some of the changes you suggested. If you could give it one more lookover, that would be appreciated!

**Review by eapache-opslevel — APPROVED** [review]

**Pull request merged**